Completion/Unix/Command/_stow

```diff
@@ -0,0 +1,83 @@
+#compdef stow chkstow
+
+#
+# A zsh completion script for GNU stow (https://www.gnu.org/software/stow/)
+#
+
+__stow_complete_packages() {
+  local IFS=$'\n'
+  local basedir="${1/\~/$HOME}"
+  _values -C "packages from $basedir" ${$( find -L "$basedir" -mindepth 1 -maxdepth 1 \( -name .git -o -name .svn \) -prune -o -type d -print 2>/dev/null | sed "s#^${basedir}/##" | sort -u ):-""}
```

Comment:
Please avoid `find`, for performance on cygwin and for correctness (filenames with spaces, etc).  In this case:

```zsh
_values -c "..." $basedir/*(-/N:t)
```

Reply:
Great, thanks for that! I didn't like this `find` too, but I couldn't find the _zsh way_ to do that.

Reply:
You're welcome. It's called [
"glob qualifiers" ([documented in zshexpn](http://zsh.sourceforge.net/Doc/Release/Expansion.html#Glob-Qualifiers)).

Reply:
Just noticed that I wrote `-c` in my previous comment. For avoidance of doubt, that was a typo: I meant to write `-C`, with a capital C as in your original.